x-pack/platform/plugins/shared/fleet/public/applications/fleet/sections/agent_policy/create_package_policy_page/components/steps/components/package_policy_input_stream.tsx

```diff
@@ -324,6 +330,72 @@ export const PackagePolicyInputStreamConfig = memo<Props>(
                   </EuiFlexItem>
                   {isShowingAdvanced ? (
                     <>
+                      {packageInfo.type === 'input' && (
+                        <EuiFlexItem>
+                          <EuiFormRow
+                            label={
+                              <FormattedMessage
+                                id="xpack.fleet.createPackagePolicy.stepConfigure.packagePolicyDataStreamTypeInputLabel"
+                                defaultMessage="Data Stream Type"
+                              />
+                            }
+                            helpText={
+                              isEditPage ? (
+                                <FormattedMessage
+                                  id="xpack.fleet.createPackagePolicy.stepConfigure.packagePolicyInputOnlyEditDataStreamTypeHelpLabel"
+                                  defaultMessage="The data stream type cannot be changed for this integration. Create a new integration policy to use a different input type."
+                                />
+                              ) : (
+                                <FormattedMessage
+                                  id="xpack.fleet.createPackagePolicy.stepConfigure.packagePolicyDataStreamTypeHelpLabel"
+                                  defaultMessage="Select a data stream type for this policy. This setting changes the name of the integration's data stream. {learnMore}."
+                                  values={{
+                                    learnMore: (
+                                      <EuiLink
+                                        href={docLinks.links.fleet.datastreamsNamingScheme}
+                                        target="_blank"
+                                      >
+                                        {i18n.translate(
+                                          'xpack.fleet.createPackagePolicy.stepConfigure.packagePolicyNamespaceHelpLearnMoreLabel',
+                                          { defaultMessage: 'Learn more' }
+                                        )}
+                                      </EuiLink>
+                                    ),
+                                  }}
+                                />
+                              )
+                            }
+                          >
+                            <EuiRadioGroup
```

Comment:
It would be probably nice to add some kind of E2E or integration test to check that this radio button works and that the expected index template is created after installing the package and adding a policy.

@juliaElastic @nchaulet wdyt? do you have recommendations about where to start adding such a test?

Reply:
We probably could have a cypress test here for that https://github.com/nchaulet/kibana/blob/9ec5275d2f6a8ca0ac3a5a37059d5cb22326fa20/x-pack/platform/plugins/shared/fleet/cypress/e2e/package_policy.cy.ts#L161